test/spec/LintingSpec.js

```diff
@@ -303,38 +600,49 @@ describe('i18n', function() {
       }
     });
 
-    var diagram = require('./diagram-with-issues.bpmn');
+    const diagram = require('./diagram-with-issues.bpmn');
 
-    modeler.importXML(diagram, function(err) {
-
-      if (err) {
-        return done(err);
-      }
+    try {
+      modeler.importXML(diagram).then(function() {
```

Comment:
What are we try/catching here? This is async so impossible to fail synchronously.

I'd rather convert this whole test case to a `async` function and use `await modeler.importXML()` to have proper error handling work out of the box.

Reply:
For async errors an appropriate patter would be `something().catch(done)`

Reply:
Thanks. 

Corrected it.

I did not go for the async test function, since we still have the callback-style `eventBus.once` invocation in the test case

Reply:
Makes sense. Thanks.